lotto/src/main/kotlin/lotto/domain/result/Judge.kt

```diff
@@ -0,0 +1,19 @@
+package lotto.domain.result
+
+import lotto.domain.Lotto.Lotto
+import lotto.domain.Lotto.Number
+
+object Judge {
+
+    fun judgeResult(purchaseLottos: List<Lotto>, winningLotto: Lotto, bonusNumber: Number): MutableMap<Int, Int> {
+        val results = mutableMapOf<Int, Int>(Pair(1, 0), Pair(2, 0), Pair(3, 0), Pair(4, 0), Pair(5, 0))
```

Comment:
```suggestion
        val results = mutableMapOf(
            *((1..5).map { Pair(it, 0) }
                .toTypedArray())
        )
```
Stream을 사용하는 방법도 있을 수 있겠네요

Reply:
`to` 중위 함수는 `Pair` 를 반환하는데, 이를 사용해보는건 어떨까요?

Reply:
좋네요 ㅎㅎ 이거는 안익숙했는데 익숙해지도록 노력해보겠습니다~